linux/user-and-file-management/linux-intro/linux-common-terminology.md

```diff
@@ -0,0 +1,60 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+---
+
+# Common Terminology
+
+---
+## Content
+
+
+Let's go over a few terms that you'll see a lot in this course.
+
+A **command** is just a piece of text that you type into the **terminal**. Think of this as an instruction that you give to your computer.
+
+Sometimes you might type some commands which are restricted. That's where the **root user** comes in. It's the unique admin user, with superpowers. Unsurprisingly, it's also called the **superuser**.
+
+The **package manager** removes the tedious process of installing or uninstalling apps. Think of a **package** as being an app.
+
+Remember the terminal? Well, this is also referred to as the **shell**. 
+
+Linux comes with an improved version of the shell, which is called **bash**[1] (**Bourne-Again SHell**). From now on, we'll be using these terms interchangeably.
```

Comment:
Should we mention the "Bourne-Again SHell" part here if we also mention it in the footnote?

Reply:
no, forgot to remove it.